recipes/jsoncpp/all/conanfile.py

```diff
@@ -66,13 +65,17 @@ def generate(self):
 
     def _patch_sources(self):
         apply_conandata_patches(self)
-        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
-                              "${jsoncpp_SOURCE_DIR}",
-                              "${JSONCPP_SOURCE_DIR}")
         if self.settings.compiler == "Visual Studio" and self.settings.compiler.version == "11":
             replace_in_file(self, os.path.join(self.source_folder, "include", "json", "value.h"),
                                   "explicit operator bool()",
                                   "operator bool()")
+        # No opt-out of ccache
+        cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
+        if Version(self.version) < "1.9.3":
+            replace_in_file(self, cmakelists, "if(CCACHE_FOUND)", "if(0)")
```

Comment:
Maybe it's better to replace a line

```find_program(CCACHE_EXECUTABLE "ccache" HINTS /usr/local/bin /opt/local/bin)```
with
```CCACHE_FOUND=0```

Reply:
here is a another solution: https://github.com/conan-io/conan-center-index/pull/14387/commits/6c72ace3bfa094407ce7db935e388cf63443a9a7